cli/cmd/login.go

```diff
@@ -129,21 +130,37 @@ func runLogin(ctx context.Context, cmd *cobra.Command) (err error) {
 
 	url := accountsURL + "?returnTo=" + localServerURL + "/callback"
 	if err := browser.OpenURL(url); err != nil {
-		fmt.Printf("Failed to open browser at %s. Please open the URL manually.\n", accountsURL)
+		fmt.Printf("Failed to open browser. Please open %s manually and paste the token below:\n", accountsURL)
+		oldState, err := term.MakeRaw(int(os.Stdin.Fd()))
```

Comment:
Without these, any type of reading (bufio scanner/reader, scanf, io.ReadAll with CTRL+D chaser) hangs after 1024 characters for me.